plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityExploredLocations.java

```diff
@@ -0,0 +1,43 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import org.bukkit.entity.Sniffer;
+
+public class EntityExploredLocations extends EntityProperty<ListTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name explored_locations
+    // @input ListTag(LocationTag)
+    // @description
+    // Controls a sniffer's explored locations.
+    // Note that if the sniffer is not in the same world as the input LocationTag(s), then the LocationTag(s) will not be added to the list of explored locations.

```

Comment:
See discussion on [Discord](https://discord.com/channels/315163488085475337/1011496047811506227/1229374560219168899) - since the JD (paper's, I.e. they specifically revised Spigot's JD to say that) only says the locations are in the same world in the setter, probably better off only promising it for the mechanism.
Property meta has a `@mechanism` key for mech-only meta now, so can use that.